uber/templates/registration/new.html

```diff
@@ -168,18 +171,19 @@
             {% if c.NUMBERED_BADGES %}
                 <td>
                     &nbsp;&nbsp;&nbsp;&nbsp;
-                    <input type="number" name="badge_num" size="5" style="width:5em" />
+                    <input type="number" name="badge_num" size="5" style="width:5em" {% if attendee.paid == c.NOT_PAID %}disabled{% endif %} />
```

Comment:
While I was in this file anyway, I made it so that if the submit button is disabled, the input element is also disabled, which is less confusion, since it no longer lets you enter data into a field you can't submit.